### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Mohitkumar33**
Here is my home page. Please review it and if found any mistakes or feel anything can be improved.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+.quiz-list p {
+    color: bisque;
+}
+
+/********footer started*************/
+
+footer {
+    margin-top: 1rem;
+    /* background-color: rgb(228, 226, 226); */
```
Comment (deepak29-git) [inline style.css]: remove commented code

**Review by deepak29-git — APPROVED** [review]
look good to me

File: style.css (hunk 1 above)
Comment (Mohitkumar33) [reply style.css]: Thanks deepak Done

**Review by ajay-mandaviya — APPROVED** [review]
every thing looks good merge it

**Pull request merged**